src/test/java/com/wikia/webdriver/testcases/mercurytests/ArticlePageTests.java

```diff
@@ -10,13 +13,16 @@
 import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.ArticlePageObject;
 
 import org.openqa.selenium.WebDriverException;
-import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 import java.io.UnsupportedEncodingException;
 import java.net.URLEncoder;
-import java.util.concurrent.TimeUnit;
 
+@Execute(
+    onWikia = MercuryWikis.MERCURY_AUTOMATION_TESTING,
+    onDevice = Device.GOOGLE_NEXUS_5,
```

Comment:
With this approach is there any way to overwrite this values from command line or some config file? Don't you think this may be misleading for someone when he/she tries to run the test on FF and it launches Chrome with mobile emulation?